packages/apollo/src/App.js

```diff
@@ -202,7 +202,7 @@ class App extends Component {
 			default:
 				break;
 		}
-		let docPrefix = 'https://www.ibm.com/docs/en/blockchain-platform/2.5.3';
+		let docPrefix = 'https://www.ibm.com/docs/en/hlf-support/1.0.0';
```

Comment:
this line shouldn't be changed, please undo

the hfl-support vs blochain-platform switching happens on line 207 if the settings are correct

Reply:
updated the code as per comment